website_crm_privacy_policy/static/src/js/scripts.js

```diff
@@ -9,28 +9,24 @@ odoo.define('website_crm_privacy_policy.crm_policy', function (require) {
     var core = require('web.core');
     var ajax = require('web.ajax');
     var _t = core._t;
+    var animation = require('web_editor.snippets.animation');
 
-    crm_policy.registry.accept_policy = snippet_animation.Class.extend({
+
+    return animation.registry.accept_policy = animation.Class.extend({
         selector: '.s_website_form',
```

Comment:
You are affecting more forms than just the `website_crm` one, right? Shouldn't this be reflected in the addon name?